src/EnergyPlus/UnitarySystem.cc

```diff
@@ -15988,7 +15984,7 @@ namespace UnitarySystems {
             SpeedNum = int(Par[5]);
             FanOpMode = int(Par[6]);
 
-            HeatingCoils::CalcMultiStageGasHeatingCoil(state, CoilIndex, SpeedRatio, CycRatio, SpeedNum, FanOpMode);
+            HeatingCoils::CalcMultiStageElectricHeatingCoil(state, CoilIndex, SpeedRatio, CycRatio, SpeedNum, FanOpMode, QActual);
```

Comment:
This line as well

Reply:
Yes, this also looks wrong.

        case DataHVACGlobals::Coil_HeatingGas_MultiStage: {
            CycRatio = Par[4];
            SpeedNum = int(Par[5]);
            FanOpMode = int(Par[6]);

            HeatingCoils::CalcMultiStageElectricHeatingCoil(state, CoilIndex, SpeedRatio, CycRatio, SpeedNum, FanOpMode, QActual, SuppHeat);

Reply:
I agree, these bugs were here before this branch.